src/sidebar/components/tag-editor.js

```diff
@@ -207,17 +224,37 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
           );
         })}
       </ul>
-      <input
-        list={datalistId}
-        onInput={handleOnInput}
-        onKeyPress={handleKeyPress}
-        onKeyUp={handleKeyUp}
-        ref={inputEl}
-        placeholder="Add tags..."
-        className="tag-editor__input"
-        type="text"
-      />
-      {suggestionsList()}
+      <span
+        id={tagEditorId}
+        className="tag-editor__combobox-wrapper"
+        ref={closeWrapperRef}
+        role="combobox"
+        aria-expanded={isOpen.toString()}
+        aria-owns={`${tagEditorId}-datalist`}
+        aria-haspopup="true"
+      >
+        <input
+          onInput={handleOnInput}
+          onKeyDown={handleKeyDown}
+          aria-controls={`${tagEditorId}-datalist`}
+          onFocus={handleFocus}
+          ref={inputEl}
+          placeholder="Add new tags"
+          className="tag-editor__input"
+          type="text"
+          autoComplete="off"
```

Comment:
The capitalization of `autoComplete` here confused me but it turns out that at least in React (I haven't tested whether Preact accepts both), [it is required](https://stackoverflow.com/questions/37503656/react-doesnt-render-autocomplete-off/37503673).

Reply:
I don't know if it's required or not, but I was getting Chrome prompts from my previously typed tags and I can't see people wanting to use that since thats exactly what the suggestions list does already and the UI looks totally different. I just want it disabled.